src/snowflake/snowpark/_internal/analyzer/datatype_mapper.py

```diff
@@ -307,6 +322,12 @@ def schema_expression(data_type: DataType, is_nullable: bool) -> str:
             raise TypeError(f"Invalid vector element type: {data_type.element_type}")
         values = [i + zero for i in range(data_type.dimension)]
         return f"{values} :: VECTOR({data_type.element_type},{data_type.dimension})"
+    if isinstance(data_type, FileType):
+        return (
+            "TO_FILE(OBJECT_CONSTRUCT('RELATIVE_PATH', 'some_new_file.jpeg', 'STAGE', '@myStage', "
+            "'STAGE_FILE_URL', 'some_new_file.jpeg', 'SIZE', 123, 'ETAG', 'xxx', 'CONTENT_TYPE', 'image/jpeg', "
+            "'LAST_MODIFIED', '2025-01-01'))"
```

Comment:
why is it hard coded with constant values?

Reply:
It's for the schema query, which just need to be a valid dummy data https://docs.snowflake.com/LIMITEDACCESS/sql-reference/functions/to_file